### DISCUSSION TITLE
Value Error: [E1010] Unable to set entity information

### DISCUSSION THREAD

**Discussion by karndeepsingh** [explosion/spaCy#9741] [Help: Coding & Implementations] [answered]
Hey,
I am facing an issue as show below in image, while setting entites in doc.ents = entites. Please, guide to remove this error.

Thanks
![image](https://user-images.githubusercontent.com/49562460/143264496-f82dfc6a-a9a9-4b25-8a5f-9d20d051b5d0.png)

**Comment by polm** [comment, marked as answer]
Like the error says, token 130 in your data has entity information that can't be used. Without your data it's not clear what the issue is, but maybe you have two labels for the same token.

Also, as mentioned in other threads, please do not post screenshots of code, copy/paste them as text.

**Reply by karndeepsingh** [reply]
How can I allow this to overwrite if a token has two labels?

**Reply by polm** [reply]
You need to filter the list so that you only have one entity per token. You can look [the source of the EntityRuler](https://github.com/explosion/spaCy/blob/a4c43e5c577d7a143ef7e2fd74ccea33aace96b7/spacy/pipeline/entityruler.py#L178) for an example of how to do this.
